sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClientBuilder.java

```diff
@@ -365,6 +408,7 @@ public KeyClientBuilder serviceVersion(KeyServiceVersion version) {
      *
      * @return The updated {@link KeyClientBuilder} object.
      */
+    @Override
```

Comment:
Setting a custom `HttpPipeline` would make it so the builder ignores some values in the `Configuration` object. Should we add a note like you did in other methods on this same class?